nodejs/week3/homework/warmup/index.js

```diff
@@ -0,0 +1,183 @@
+const express = require('express');
+const app = express();
+
+// Parse URL-encoded bodies (as sent by HTML forms)
+app.use(express.urlencoded({ extended: true }));
+// Parse JSON bodies (as sent by API clients)
+app.use(express.json());
+
+// get methods:
+app.get('/calculator', (req, res) => {
```

Comment:
While what you did works, it is too complicated to include everything in a single route. I suggest you create 4 separate routes for the addition/subtraction/multiplication/division. Eg:
```
app.get("/calculator/addition", (req, res)
```

Reply:
> Very good work 👍 Only thing you should fix, is splitting the routes and not having all 4 features(addition/subtraction..) in the same route, have a look at my comment :)

yes, it increass the same code line repeated but according to your suggestion i have splitted them into four separate routs.  thank you

Reply:
> While what you did works, it is too complicated to include everything in a single route. I suggest you create 4 separate routes for the addition/subtraction/multiplication/division. Eg:
> 
> ```
> app.get("/calculator/addition", (req, res)
> ```

i refactored it according to your suggestions